### PR TITLE
Update driver page with more information

### PR DESCRIPTION
**Pull request by saad-ali**
I looked at each drivers linked code or documentation to generate this. Instead of having each driver verify this, let's get it merged, and they can make whatever changes they want for their driver afterwards.

/assign @msau42

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -5,48 +5,50 @@ The following are a set of CSI driver which can be used with Kubernetes:
...
+[NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | No | Yes
+[Nutanix](https://portal.nutanix.com/#/page/docs/details?targetId=CSI-Volume-Driver:CSI-Volume-Driver) | `"com.nutanix.csi"` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Nutanix | Persistent | ? | ? | ? | ?
+[OpenSDS](https://github.com/opensds/nbp/tree/master/csi) | `csi-opensdsplugin` | v1.0 | A Container Storage Interface (CSI) Driver for  [OpenSDS]((https://www.opensds.io/)) | Persistent | Read/Write Single Pod | Yes | Yes | Yes
+[Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | v0.3 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | ? | ? | ? | ?
```
Comment (msau42) [inline book/src/drivers.md]: This row is off

File: book/src/drivers.md [hunk 2]
```diff
@@ -5,48 +5,50 @@ The following are a set of CSI driver which can be used with Kubernetes:
...
+[QingCloud CSI](https://github.com/yunify/qingcloud-csi)| `csi.qingcloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for QingCloud Block Storage | Persistent | Read/Write Single Pod | Yes | No | No
+[QingStor CSI](https://github.com/yunify/qingstor-csi) | `csi-neonsan` | v0.3 | A Container Storage Interface (CSI) Driver for NeonSAN storage system | Persistent | Read/Write Single Pod | Yes | No | Yes
+[Quobyte](https://github.com/quobyte/quobyte-csi) | `quobyte-csi` | v0.2 | A Container Storage Interface (CSI) Driver for Quobyte | Persistent | Read/Write Multiple Pods | Yes | No | No
+[ScaleIO](https://github.com/thecodeteam/csi-scaleio) | `com.thecodeteam.scaleio` | v0.2 | A Container Storage Interface (CSI) Driver for DellEMC ScaleIO | Persistent | Read/Write Single Pod | Yes | No | No
+[StorageOS](https://github.com/storageos/charts/blob/master/stable/storageos/README-CSI.md) | v1.0 | A Container Storage Interface (CSI) Driver for [StorageOS](https://storageos.com/) | Persistent | Read/Write Multiple Pods | Yes | No | No
```
Comment (msau42) [inline book/src/drivers.md]: This row is off

**Comment by saad-ali** [comment]
PTAL

**Pull request merged**